Makefile

```
@@ -1,42 +1,36 @@
-.PHONY: help all clean test build release lint fmt check-fmt markdownlint nixie
+.PHONY: be fe openapi gen docker-up docker-down fmt lint test check-fmt markdownlint
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)